lib/src/render/link_menu/link_menu.dart

```diff
@@ -111,33 +114,42 @@ class _LinkMenuState extends State<LinkMenu> {
   }
 
   Widget _buildInput() {
-    return TextField(
-      focusNode: _focusNode,
-      style: TextStyle(
-        fontSize: style?.textStyle?.fontSize,
-      ),
-      textAlign: TextAlign.left,
-      controller: _textEditingController,
-      onSubmitted: widget.onSubmitted,
-      decoration: InputDecoration(
-        hintText: 'URL',
-        hintStyle: TextStyle(
+    return RawKeyboardListener(
+      focusNode: FocusNode(),
+      onKey: (key) {
+        if (key is! RawKeyDownEvent) return;
```

Comment:
```suggestion
        if (key is! RawKeyDownEvent) {
          return;
        }
```
IIRC, Effective Dart says to use curly braces.